llvm/test/Transforms/SimplifyCFG/mmra.ll

```diff
@@ -0,0 +1,53 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt -passes=simplifycfg -simplifycfg-require-and-preserve-domtree=1 -S %s | FileCheck %s
+
+; TODO: Should this work?
+define i32 @test1(i1 zeroext %flag, i32 %x, ptr %y) {
```

Comment:
This (IMO rightfully) doesn't work. I remember making this change because otherwise, almost all MMRAs ended up dropped because SimplifyCFG (AFAIK) canonicalizes the CFG so it kicks in a ton.

Thoughts @sameerds ?